### PR TITLE
Add a post-action hook to allow for GCE deprovisioning

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

@smarterclayton PTAL

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
https://ci.openshift.redhat.com/jenkins/job/test_branch_origin_extended_conformance_gce/83/

File: config/test_cases/test_branch_origin_extended_conformance_gce.yml [hunk 1]
```diff
@@ -123,22 +115,23 @@ extensions:
...
-          export INSTANCE_PREFIX="prtest-${hashed_identifier:0:7}-${BUILD_NUMBER}"
-          ../../bin/local.sh ansible-playbook playbooks/terminate.yaml
         }
-        trap cleanup EXIT
+        trap gather EXIT
 
         make test-extended SUITE="conformance"
+  post_actions:
+    - type: "script"
+      title: "deprovsion test cluster"
```
Comment (smarterclayton) [inline config/test_cases/test_branch_origin_extended_conformance_gce.yml]: You spelled deprovision wrong :)

Comment (stevekuznetsov) [inline config/test_cases/test_branch_origin_extended_conformance_gce.yml]: This is such a consistent spelling mistake, it's gone beyond embarrassing ...

**Comment by stevekuznetsov** [comment]
So in that run we actually hit the `/healthz` flake and the post-hook kicked in as expected -- @smarterclayton I think this is probably good to go

**Pull request merged**